packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -245,4 +253,35 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
       allowMultiLogin: this.$config?.allowMultiLogin ?? true,
     };
   }
+
+  /**
+   * Called when we finish registration to silently handle the errors
+   */
+  private async defaultReLogin(): Promise<StationReLoginResponse | void> {
```

Comment:
Here, we silently handle the error (usually because agent is not logged in) - this is being done in agentx also, ie they always call relogin after registration.